BangazonWebApp/BangazonWebApp/Models/Computers.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonWebApp.Models
+{
+    public class Computers
+    {
+        public int Id { get; set; }
+
+        [Display(Name = "Date Purchased")]
+        public DateTime DatePurchased { get; set; }
+        [Display(Name = "Decommissioned Date")]
+        [Required]
+        public DateTime DecommissionedDate { get; set; }
```

Comment:
`DecommissionedDate` should not be a required property- what if a computer hasn't been decommissioned yet? (I also noticed that in your database, this property is actually set to `NOT NULL`- that needs to change too. A computer that is currently in use should be able to have a null decommissioned date.)

In general, you should put the `[Required]` annotation over anything that is specified as `NOT NULL` in your database, but you also want to make sure that those restrictions make sense in the context of the business.